src/AccessibilityTranslator.js

```diff
@@ -137,7 +158,6 @@ export default class AccessibilityTranslator extends React.Component {
     if (!displayObject.visible && props['aria-hidden'] === undefined) {
       props['aria-hidden'] = true;
     }
-
```

Comment:
Nit: please keep this blank line.  Removing it makes it seem likely it's related to just the above if statement, though not part of its condition, instead of what the whole function has been working towards.

Reply:
updated